src/App.jsx

```diff
@@ -1,10 +1,27 @@
+import { BrowserRouter } from 'react-router-dom';
 import React from 'react';
 import styled from 'styled-components';
+import NavBar from './NavBar';
 
-const Title = styled.h1`
-  color: blue;
+// Styling section
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
```

Comment:
this is the default you should be able to remove this line

Reply:
Thanks, i didnt know thios :smile: